NanoVNASaver/Windows/TDR.py

```diff
@@ -34,7 +34,7 @@ def __init__(self, app: QtWidgets.QWidget):
         super().__init__()
         self.app = app
 
-        self.td = []
+        self.td = None
```

Comment:
```suggestion
        self.td = []
```
I prefer variable initialization with target type over "None".
Both will evaluate to false in if statement. But if you do something like
for elem in self.td:
   ....
None is a source of error.

Reply:
it's fine to initialize empty objects. However, td is at least sometimes a numpy array and not a list. That means e.g. `if self.tdrWindow.td` raises a ValueError when td has 2 elements and more:
```
if np.empty(0):
	print("empty arrays evaluate to true but raise a DeprecationWarning and will raise an error in the future")
if np.zeros((1,)):
	print("unfortunately, this just passes?")
if np.zeros((2,)):
	print("arrays with more than 1 element raise a ValueError, so I get never printed")
```

You can `self.td = np.empty(0)` and then check if td is non-empty:
```
td = np.zeros((2,))
if td.size != 0:
	print("success!")
```